app/src/main/java/com/aileyzhang/musicapp/MainActivity.java

```diff
@@ -25,6 +25,8 @@
         ActivityCompat.OnRequestPermissionsResultCallback {
 
     private static final int SDCARD_PERMISSION_REQUEST_CODE = 1;
+    private static final int SONGS_PAGE_POSITION = 0;
```

Comment:
Great use of constants, just a nitpick but it might make more sense if these were declared in the view pager adapter since you would only use it when you're using the adapter which defines which view is shown for each index :) I see that type of relationship of these constants being owned by the view adapter